x/staking/types/delegation_test.go

```diff
@@ -1,11 +1,13 @@
 package types
```

Comment:
It'd be nice to build tests as a separate `types_test` package. Not a blocker though